test/functional/apps/visualize/_tsvb_time_series.ts

```diff
@@ -86,6 +86,33 @@ export default function({ getPageObjects, getService }: FtrProviderContext) {
         const actualCount = await visualBuilder.getRhythmChartLegendValue();
         expect(actualCount).to.be(expectedLegendValue);
       });
+
+      it('should show the correct count in the legend with percent formatter', async () => {
```

Comment:
What if we create a new `describe `block for all tests started from _'should show the correct count ...'_? 
It will be more readable for other developers, also `await visualBuilder.clickSeriesOption(); `can be moved into beforeEach block